cmd/validate_editorconfig.go

```diff
@@ -179,7 +179,6 @@ func checkVersion(config config.Config) error {
 
 // addPersistentFlags adds flags to the root command
 func addPersistentFlags(cmd *cobra.Command) {
-	cmd.PersistentFlags().StringSliceVar(&configFilePaths, "config", defaultConfigFileNames, "Paths to the configuration files")
```

Comment:
Where do we set the config File paths now? 

This variable was used if the user wanted to override it.